airflow/providers/slack/operators/slack.py

```diff
@@ -161,15 +161,25 @@ class SlackAPIFileOperator(SlackAPIOperator):
 
     .. code-block:: python
 
+        # Send file with filename and filetype
         slack = SlackAPIFileOperator(
-            task_id="slack_file_upload",
-            dag=dag,
-            slack_conn_id="slack",
-            channel="#general",
-            initial_comment="Hello World!",
-            filename="hello_world.csv",
-            filetype="csv",
-            content="hello,world,csv,file",
+           task_id="slack_file_upload",
```

Comment:
This is indented by three spaces?

Reply:
@uranusjr Thats the sample code for the docs, I can fix it.

Reply:
Fixed.